### ISSUE TITLE
Instructions are hard to follow

### ISSUE THREAD

**Issue by JohnReedLOL** [lihaoyi/Scalite#9]
I am a new Scala programmer. I use IntelliJ. Inside my Scala project folder, there is no build folder. There is no folder named "project" and no folder named "build". How do you set up this tool?

**Comment by nightscape** [comment]
The instructions assume familiarity with [SBT](http://www.scala-sbt.org/), the most commonly used Scala Build Tool, which has this kind of project structure.
Unfortunately, you won't have much chance of getting Scalite to work in IntelliJ...
IntelliJ ships its own parsers which don't have any clue what to do with Scalite files ;)
As much as I love the lightweight syntax of Scalite, for you as a Scala novice I would very much recommend you to stick with ordinary Scala syntax otherwise you won't have any IDE support.

**Comment by JohnReedLOL** [comment]
I appreciate your honesty. There's a mention of your ad-on here... 
https://issues.scala-lang.org/browse/SI-9512

**Comment by yawaramin** [comment]
@JohnReedLOL OK to close this?
